cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -142,3 +148,28 @@ func SetDefaultsEtcdSelfHosted(obj *MasterConfiguration) {
 		obj.Etcd.SelfHosted.CertificatesDir = DefaultEtcdCertDir
 	}
 }
+
+// SetDefaults_KubeletConfiguration assigns default values to kubelet
+func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
```

Comment:
This isn't autogenerated in zz_generated.defaults.go. Any ideds?

Reply:
It should be manually called from `SetDefaults_MasterConfiguration`